RLTest/__main__.py

```diff
@@ -489,6 +489,10 @@ def __init__(self):
                     print(Colors.Bred('Using `--module` multiple time implies that you specify the `--module-args` in the the same number'))
                     sys.exit(1)
 
+        if self.args.no_output_catch and self.args.parallelism > 1:
+            print(Colors.Bred('No output catch can not be combine with parallel test execution.'))
```

Comment:
```suggestion
            print(Colors.Bred('No output catch can not be combined with parallel test execution.'))
```

Reply:
This sentence is hard to digest due to two negations. How about `No output catch can be combined with parallel test execution.` or `Output catch cannot be combined with parallel test execution`?

Reply:
Or
`"No output catch" cannot be combined with parallel test execution.`

Reply:
How about now?

Reply:
:-)